tests/pixels/test_encoder_pylibjpeg.py

```diff
@@ -992,8 +993,8 @@ def test_arr_u4_spp1(self):
         }
         atols = [1, 1, 1, 1, 2, 2, 2, 2]
         atols.extend([2, 2, 2, 4, 5, 8, 10, 16])
-        atols.extend([23, 31, 52, 63, 116, 2928854, 7404089, 10080970])
-        for bits_stored, atol in zip(range(1, 25), atols):
+        atols.extend([23, 31, 52, 63, 122, 2928854, 7404089, 10080970])
```

Comment:
What has changed here (and below)?

Reply:
I have a vague guess that MacOS is using a different RNG which is affecting the lossy results, but really I have no idea. It's not so important what the actual values are, as long as they're reasonable and don't require constant updates (hah).

I've also cut back on the tested number of bits because > ~20 is pretty terrible for lossy encoding with OpenJPEG (up to 24 works OK for lossless). I kept the `atol` list as-is to show how terrible they get.

Reply:
Ok, thanks!